### PR TITLE
Added extra steps to update the Cudnnn DLL found in the Torch packages

### PR DESCRIPTION
**Pull request by Keerigan45**
I added extra steps to update the Cudnnn DLL found in the Torch package because it wasn't optimised or didn't use the lastest version. So manually updating it can speed up iteration but the result might differ from each card. Exemple i passed from 3 it/s to a steady 20 it/s.

## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [x] Documentation Update
- [ ] Community Node Submission

## Have you discussed this change with the InvokeAI team?
- [x] Yes
- [ ] No, because:

      
## Have you updated all relevant documentation?
- [x] Yes
- [ ] No

## Related Tickets & Documents

- Related Issue #
- Closes #

## Added/updated tests?

- [x] Yes
- [ ] No : _please replace this line with details on why tests
      have not been included_

### CODE REVIEW

**Comment by Malrama** [comment]
Maybe also add if one should download the CUDA 11.X or CUDA 12.X version of CUDNN because that is something one is asked to choose from before downloading.
I assumed it was CUDA 11.X, am I correct?

**Comment by Keerigan45** [comment]
https://docs.nvidia.com/deeplearning/cudnn/support-matrix/index.html May add this link and ask to just download the 11x version if their architecture doesn't show up. Or they can try both if they got time to see which version works better.

**Review by Millu — APPROVED** [review]
Fixed some types and updated readability, looks good to me!

File: docs/installation/030_INSTALL_CUDA_AND_ROCM.md [hunk 1]
```diff
@@ -57,6 +57,27 @@ familiar with containerization technologies such as Docker.
...
+### cuDNN Installation for 40/30 Series Optimization* (Optional)
+
+1. Find the InvokeAI folder
+2. Click on .venv folder - e.g., YourInvokeFolderHere\\.venv
+3. Click on Lib folder - e.g., YourInvokeFolderHere\\.venv\Lib
+4. Click on site-packages folder - e.g., YourInvokeFolderHere\\.venv\Lib\site-packages
+5. Click on Torch directory - e.g., YourInvokeFolderHere\InvokeAI\\.venv\Lib\site-packages\torch
+6. Click on the lib folder - e.g., YourInvokeFolderHere\\.venv\Lib\site-packages\torch\lib
+7. Copy everything inside the folder and save it elsewhere as a backup.
+8. Go to https://developer.nvidia.com/cudnn
```
Comment (Millu) [inline docs/installation/030_INSTALL_CUDA_AND_ROCM.md:69]: Add the link to 11.x/12.x

**Review by Millu — CHANGES_REQUESTED** [review]

**Review by Millu — APPROVED** [review]
Looks great, thank you for adding this!

**Pull request merged**